src/Classes/Providers/ChocoParcelProvider.ps1

```diff
@@ -96,7 +96,7 @@ class ChocoParcelProvider : ParcelProvider
         return "--version $($_package.Version)"
     }
 
-    [string] GetSourceArgument([ParcelPackage]$_package)
+    [array] GetSourceArgument([ParcelPackage]$_package)
```

Comment:
Keep this as a `[string]`. This function just just return a single argument that can be appended onto the install call.

Reply:
Done